spec/vector_spec.rb

```diff
@@ -177,11 +177,12 @@
               [:c,:one,:bar],
               [:c,:one,:baz],
               [:c,:two,:foo],
-              [:c,:two,:bar]
+              [:c,:two,:bar],
+              [:d,:one,:foo]
             ]
```

Comment:
Had to add `[:d, :one, :foo]` to test a special case i.e. single index exist under second layer.